assets/js/EmbedMaker.js

```diff
@@ -149,16 +149,21 @@ export default class EmbedMaker extends Reactive {
 
     let maxBytes = dv.byteLength;
     let pos = 2;
+    var littleEndian = false;
```

Comment:
"var" is not what the cool kids use anymore, please replace it with `let` and I would also sort the variables by name.